xml/systemd.xml

```diff
@@ -2105,6 +2105,192 @@ Apr 04 16:52:26 g144 start_apache2[3088]: httpd2-prefork: Syntax error on line
+<option>OnUnitActiveSec</option> triggers the service 24 hours after the service
+has been activated (that is, the timer will trigger the service once a day).
+Finally the option <option>WantedBy</option> specifies that the timer should
+start when the system has reached the multi-user target.
+</para>
+<para>
+Instead of a monotonic timer, you can specify a realtime one using the option
+<option>OnCalendar</option>. The following realtime timer definition triggers
+the related service unit once a week, starting on Monday at 12:00.
+</para>
+<screen>[Timer]
+OnCalendar=weekly
+Persistent=true</screen>
+<para>
+The option <option>Persistent=true</option> indicates that the service will be
+triggered immediately after the timer activation if the timer missed the last
+start time (for example, due to the system being powered off).
+</para>
+<para>
+The option <option>OnCalendar</option> can also be used to define specific dates
+times for triggering a service using the following format: <literal>DayOfWeek
+Year-Month-Day Hour:Minute:Second</literal>. The examle below triggers a service
+at 5am every day:
+</para>
+<screen>OnCalendar=*-*-* 5:00:00</screen>
+<para>
+You can use an asterisk to specify any value, and commas to list possible
+values. Use two values separated by .. to indicate a contiguous range. The
+following example triggers a service at 6pm on Friday of every month: 
+</para>
+<screen>OnCalendar=Fri *-*-1..7 18:00:00</screen>
+<para>
+To trigger a service at different times, you can specify several
+<option>OnCalendar</option> entries:
+</para>
+<screen>OnCalendar=Mon..Fri 10:00
+OnCalendar=Sat,Sun 22:00</screen>
+<para>
+ In the example above, a service is triggered at 10am on week days and at
+ 10pm on weekends.
+ </para>
+<para>
+When you are done editing the timer unit file, save it under the
+name <filename>foo.timer</filename> in the
+<filename>/etc/systemd/system/</filename> directory. To check the correctness of
+the created unit files, run the following command:
+</para>
+<screen>&prompt.sudo; systemd-analyze verify /etc/systemd/system/foo.*</screen>
+<para>
+If the command returns no output, the files have passed the verification
+successfully.
+</para>
+<para>
+To start the timer, use the command <command>sudo systemctl start
+foo.timer</command>. To enable the timer on boot, run the command <command>sudo
+systemctl enable foo.timer</command>.
+</para>
+  </sect2>
+  <sect2 xml:id="sec-boot-systemd-timer-manage">
+  <title>Managing &systemd; timers</title>
+  <para>
+  Since timers are treated as regular &systemd; units, you can manage them using
+  <command>systemctl</command>. You can start a timer with <command>systemctl
+  start</command>, enable a timer with <command>systemctl enable</command>, and
+  so on. In addition to that, you can list all active timers using the command
+  <command>systemctl list-timers</command>. And to list all timers,
```

Comment:
```suggestion
  <command>systemctl list-timers</command>. To list all timers,
```

Reply:
There is a general rule not to start sentences with the word "and", at least in formal contexts.

https://www.prdaily.com/why-you-should-stop-starting-sentences-with-and-or-but/